docs/devenv_cloud.md

```diff
@@ -0,0 +1,142 @@
+# MicroShift Development Environment in Cloud
+
+## Introduction
+
+The MicroShift development environment should be set up on a **local workstation** as a collection of virtual machines.
+* The [development virtual machine](./devenv_rhel8.md#create-development-virtual-machine) used for compiling and running the MicroShift binaries and building [RHEL for Edge Installer ISO](./rhel4edge_iso.md#build-rhel-for-edge-installer-iso) images
+* The [RHEL for Edge virtual machine](./rhel4edge_iso.md#install-microshift-for-edge) used for running MicroShift in a clean, production-like environment
+* The [Ansible server virtual machine](../ansible/README.md) used for running performance benchmark automation scripts
+
+However, depending on a local workstation hardware configuration and operating system, it may not always be possible to set up the desired development environment, as in the following cases.
+* Different hardware platform setup (i.e. `x86_64` local hardware vs `aarch64` development environment or vise versa)
+* Lack of hardware resources (i.e. not enough RAM or local storage)
+
+In such cases, it is recommended to use a public cloud for setting up the MicroShift Development environment, as described in the remainder of this document.
+
+## Public Cloud Configuration
+
+The goal is to use public cloud resources for setting up the following configuration.
+* The development machine running on a cloud virtual instance
+* The RHEL for Edge virtual machine running as a **nested** virtual instance on the development host
+* The Ansible server virtual machine running as a **nested**  virtual instance on the development host
```

Comment:
Why do we need/recommend another VM for running Ansible?
BTW. [According to Ansible's docs it's "control node"](https://docs.ansible.com/ansible/latest/network/getting_started/basic_concepts.html#control-node), not server

Reply:
I see that's using 2vcpu and 2gb, so that's MicroShift VM for performance testing? Maybe we should call it that

Reply:
No, it's a host we use for starting the Ansible scripts from. We need a separate one because of dependencies, etc.  required for Ansible jobs.

Reply:
For the naming, it's called Ansible server in the https://github.com/openshift/microshift/blob/main/ansible/README.md doc. I'll change it to control node - good catch.